### ISSUE TITLE
any comped subscriptions expiring in March?

### ISSUE THREAD

**Issue by davidmooreppf** [yescomment/bh-flaminghydra-ghostconfig#8]
Per Shirley's email to us with subject heading Expiration thing? Any tasks to-do?

**Comment by unitof** [comment]
Yes. Was wondering if we could dedicate some tomorrow time to that.

Basically everyone still has infinite subscriptions. Want to write a script to add expirations based on donation amount.

A couple we manually entered with a month expiration so want to see if Outpost upsold them.

@hiyasw

**Comment by hiyasw** [comment]
Would love to tackle tomorrow!! We haven't written Outpost Auto-copy for upselling, so I don't think people have been sent anything. But will do asap!!

**Comment by davidmooreppf** [comment]
Yeah seems like we should get to that tmw.

**Comment by davidmooreppf** [comment]
Just a quick confirmation that Jacob will be looking today at the approx. 20 emails (those who gave less than $36) who we need to tag with a membership end date in our Ghost, and then activate Outpost, to see if Outpost worked to convert any of them.

**Comment by hiyasw** [comment]
Are those the "random issue" and "printable gift pak" folks from KS? who are currently "free subscribers" (but haven't been receiving any emails from us?)

**Comment by unitof** [comment]
Correction: 2500 emails we need to tag with an expiration date!

There are 20 or so we tagged manually, back when we were putting stragglers
in (like DonorBox folks who gave $3 or $10).

Comma,
Jacob Ford
Designer <https://jacobford.com> About <https://nocomment.llc> Town
<https://geistlist.email>

**Comment by hiyasw** [comment]
Thank you for tackling this! Pls explain what you set up with auto responder on Outpost when u get a chance...

Ok not sure where this goes in auto responder world but... here is the copy we want for expiring subscriptions:

Re: Return to Flaming Hydra 

Hello! We're writing to let you know that your Flaming Hydra newsletter will stop arriving on %SUBSCRIPTION_EXPIRATION_DATE%. A disaster looms!

To continue your subscription, and support real writers and artists with interesting and valuable things to share, please log in to [flaminghydra.com](http://flaminghydra.com/). It will take only a moment.

Thank you for your support of our project. It means the world to us.


Fierily yours,

The Editors, Writers and Artists of Flaming Hydra

**Comment by unitof** [comment]
We want these to be a bit different, as they are going out to *backers &
donors* who prepaid for their subscription. They are not typical
expirations, as the user never set an expiration, and they sent us money
via a third party platform before we even had subscriptions (Kickstarter
for most, DonorBox for some).

Want to emphasize less that it’s expiring or they did something wrong, and
explain that because they backed us via Kickstarter or DonorBox before we
launched, we comped their first year¹, and now we need a payment on file to
renew.

If they renew, the get to keep their FOUNDING MEMBER static and price of
$25/year. If they don’t, they’d have to rejoin at $3/month or $36/year.

You can see the two I set up for now, at 7 days and 24 hours before comp
expiration: https://flaminghydracom.outpost.pub/home#/email-templates (top
two)

Comma,
Jacob Ford
Designer <https://jacobford.com> About <https://nocomment.llc> Town
<https://geistlist.email>

**Comment by hiyasw** [comment]
Actually it will be $36 for next year for everyone, even founding members, as far as we know! We will adjust the expiration text... Thank you

**Comment by hiyasw** [comment]
Will these outpost auto replies about entering billing information (the ones you set up) go out this month or anytime soon?
